src/passthrough/sync_io.rs

```diff
@@ -222,8 +222,9 @@ impl<D: AsyncDrive> PassthroughFs<D> {
             e
         })?;
 
-        if let Some(h) = handle {
-            let hd = self.handle_map.get(h, inode)?;
+        if self.no_open.load(Ordering::Relaxed) && handle.is_some() {
+            // Safe as we just checked handle
+            let hd = self.handle_map.get(handle.unwrap(), inode)?;
```

Comment:
Why are we getting a handle with no_open? Without open, how does the fuse client get the file handle?

Reply:
kernel send request with fuse_file->fh as 0 (as there's no handle returned from fuse server), and it doesn't check if no_open is used or not, looks like kernel depends on fuse server to handle this case.